src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -119,4 +121,21 @@ private String getFrontendServiceUrl() {
         return frontendServiceUrl;
     }
 
+    private String getBackEndServiceUrl(HttpServletRequest request) {
```

Comment:
This function already exist in EI Frontend Webserver controller class. Is this a duplication of the that function?

Reply:
A similar function exists in EIRequestsController by the name 'getEIBackendSubscriptionAddress'. You have modified that class and function but maybe you can reuse it somehow.

Reply:
Improvements and new tests of those has been added.